app/domain/authentication/authn_jwt/authenticator.rb

```diff
@@ -13,7 +13,7 @@ module AuthnJwt
     ) do
 
       def call
-        create_authentication_parameters
+        authentication_parameters
```

Comment:
i would actually leave this as `create_authentication_parameters` and add the method
```
def create_authentication_parameters
  authentication_parameters
end
```

this way the `call` function tells the story

Reply:
Done